style.css

```diff
@@ -446,3 +472,566 @@ button:hover {
   height: 40px;
   list-style: none;
 }
+
+/* media queries */
+@media (min-width: 550px) and (max-width: 900px) {
+  .supporting-text {
+    margin-left: 45px;
+  }
+
+  .links {
+    margin-left: 45px;
+  }
+
+  .headline {
+    background-image: url(images/Illustration.png);
+    background-size: 250px;
+    background-position: 100%;
+  }
+
+  .introduction-text {
+    font-size: 30px;
+  }
+
+  .description {
+    width: 500px;
+    font-size: 20px;
+  }
+
+  /* work section */
+  .first-project-container {
+    display: grid;
+    justify-content: center;
+  }
+
+  .recent-text {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+  }
+
+  .line {
+    width: 680px;
+    height: 0;
+  }
+
+  .orange-image {
+    display: none;
+  }
+
+  .desktop-orange-image {
+    display: block;
+    margin-left: 20px;
+  }
+
+  .orange-container {
+    margin-left: 20px;
+  }
+
+  h4 {
+    margin-top: 10px;
+  }
+
+  .text {
+    width: 650px;
+    font-size: 20px;
+  }
+
+  .container {
+    display: grid;
+    grid-template-columns: 369px 369px;
+    justify-content: center;
+    grid-gap: 30px;
+  }
+
+  .project-button {
+    width: 369px;
+  }
+
+  .about-me {
+    text-align: center;
+  }
+
```

Comment:
- Please try to properly align the sections so that it looks similar to the Figma design.
- Notice: The line before the section with languages, framework, and skills, looked centered, but I noticed you made use of `padding-left: 24px;` on the entire `About me section`, I would suggest you make use of width size less than `90%` and then use a `margin: auto` to make it center.

##### Figma design
![Screenshot from 2022-06-02 17-55-27](https://user-images.githubusercontent.com/58771507/171685477-5869253f-f972-41e8-83aa-4acab0be457f.png)

##### Your design 
![Screenshot from 2022-06-02 17-56-04](https://user-images.githubusercontent.com/58771507/171685530-f36294bb-7b28-41fd-8027-7119007fd8e5.png)

**Note:** Feel free to reach out to me on slack `@kanu mike chibundu`  if you find this feedback confusing